### DISCUSSION TITLE
Best Practices for Deploying the Back End of a VSCode Plugin Utilizing GenAI

### DISCUSSION THREAD

**Discussion by jbmck** [microsoft/vscode-discussions#700] [Extension Development QnA] [answered]
Can anyone point me in the direction of some best practices/architecture diagrams for Deploying the Back End of a VSCode Plugin that will utilize GenAI by calling an LLM?

**Comment by isidorn** [comment, marked as answer]
Start here https://code.visualstudio.com/api/get-started/your-first-extension
After you have a simple extension adding GenAI by calling an LLM should be an easy next step.
